go/arrow/array/list.go

```diff
@@ -422,6 +431,10 @@ func (b *baseListBuilder) Append(v bool) {
 	b.appendNextOffset()
 }
 
+func (b *baseListBuilder) AppendWithSize(v bool, list_size int) {
```

Comment:
the typical style here would be `AppendWithSize(v bool, _ int)` using the underscore since you aren't using the `int` argument in this impl

Reply:
Fixing.